Kickstarter-iOS/Features/PledgeView/Views/PledgeViewCTAContainerView.swift

```diff
@@ -292,3 +304,55 @@ private func attributedTermsText() -> NSAttributedString? {
 
   return checkoutAttributedLink(with: string)
 }
+
+private func pledgeImmediatelyText() -> NSAttributedString? {
+  let rawText = Strings.Your_payment_method_will_be_charged()
+
+  guard let text = try? NSMutableAttributedString(
+    data: Data(rawText.utf8),
+    options: [
+      .documentType: NSAttributedString.DocumentType.html,
+      .characterEncoding: String.Encoding.utf8.rawValue
+    ],
+    documentAttributes: nil
+  ) else {
+    return nil
+  }
```

Comment:
Will the helper method `simpleHtmlAttributedString` do what you need, here? Or is the custom implementation required?

Reply:
Oh nice find! I looked for something like this but clearly not hard enough - I really wasn't expecting us to already handle strings like this.